config/jobs/kubernetes-sigs/container-object-storage-interface/container-object-storage-interface.yaml

```diff
@@ -16,19 +20,29 @@ presubmits:
         - runner.sh
         args:
         - make
+        - build
         resources:
           limits:
             cpu: 2
             memory: 4Gi
           requests:
             cpu: 2
             memory: 4Gi
+        env:
+        - name: USE_DOCKERIZED_BUILD
+          value: "true"
+        # docker-in-docker needs privileged mode
+        securityContext:
+          privileged: true
```

Comment:
COSI's build uses docker, and COSI's e2e tests use KinD. I think this requires this configuration (as well as the 2 dind and service account labels above, but I'm not 100% certain. I'm trying to follow what I see in other prow configs here to make sure we are doing this the right way.